core/templates/dev/head/css/oppia.css

```diff
@@ -356,6 +356,120 @@ md-content.oppia-profile-portfolio-container {
   TODO(sll): This is ugly. We should try not to have it, perhaps after
   angular-material has been upgraded.
 */
+
+
+.tooltip {
```

Comment:
Hi @jblair87, lots of issues here:
- Please ensure you take a look at the surrounding file before making wide-ranging changes. The addition of this code causes the comments above this line to lose their context since they are separated from the code they refer to.
- The code here does not follow the existing style (which involves indenting by 2 spaces, not 4, and alphabetizing the properties within each selector). There are also a lot of superfluous newlines. Please see the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide).
- The classes here are too widely scoped. They will apply to all tooltips throughout the entire site. Is this actually intended? If so, did you check each case carefully to ensure that no part of the existing UI is negatively affected?
- Finally, does this code need to be added at all? I thought the idea was to replace the tooltip with a Bootstrap popover -- please take a look at how it's done in the Profile page. A small change like the one this PR is meant to fix should not, as a general rule, require a lot of additional code, since this imposes a maintenance burden.